search/src/main/java/edu/artic/search/SearchAudioDetailViewModel.kt

```diff
@@ -76,6 +85,13 @@ class SearchAudioDetailViewModel @Inject constructor(private val analyticsTracke
                 .bindTo(playAudioVisible)
                 .disposedBy(disposeBag)
 
+        articObjectObservable
+                .filterFlatMap({it.gallery != null} , {it.gallery!!})
+                .filter { it.number != null }
+                .map { "Gallery ${it.number.toString()}" } // TODO: use localizer
```

Comment:
We can at least use a string resource here.

Reply:
updated to pull generation of this full field into Fragment under protest